src/runtime/virtcontainers/sandbox.go

```diff
@@ -803,6 +809,12 @@ func (s *Sandbox) createNetwork(ctx context.Context) error {
 	span, ctx := katatrace.Trace(ctx, s.Logger(), "createNetwork", sandboxTracingTags, map[string]string{"sandbox_id": s.id})
 	defer span.End()
 
+	network, err := NewNetwork(&s.config.NetworkConfig)
```

Comment:
Remind me; we are setting the sandbox's network in both newSandbox and createNetwork -- are these executed in differing flows, or is this duplicated?

Reply:
It's a dup, carried over from the switch between netns to the `Network` structure and then interface.
I pushed an additional commit to simplify `createNetwork()`.